pkgs/by-name/pl/plymouth-matrix-theme/package.nix

```
@@ -0,0 +1,46 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  unstableGitUpdater,
+}:
+stdenvNoCC.mkDerivation {
```

```suggestion
}:

stdenvNoCC.mkDerivation {
```
Skip a line after arguments.